pkg/nativeimgutil/fuzz_test.go

```diff
@@ -0,0 +1,36 @@
+package nativeimgutil
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func FuzzConvertToRaw(f *testing.F) {
+	f.Fuzz(func(t *testing.T, imgData []byte, withBacking, specifySize bool) {
+		if len(imgData) < 3 {
+			return
+		}
+		imgDir, err := os.MkdirTemp("", "fuzz-images-")
```

Comment:
I think you want to use `testing.TempDir`, or defer the cleanup

Reply:
https://github.com/lima-vm/lima/pull/2479/commits/e0723fd8a7c6660a89dc72e49fb2c5436a24c247